bundles/org.openhab.binding.valloxmv/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -18,6 +18,7 @@
 			<channel id="tempincomingbeforeheating" typeId="tempincomingbeforeheating"/>
 			<channel id="tempincoming" typeId="tempincoming"/>
 			<channel id="humidity" typeId="humidity"/>
+            <channel id="co2" typeId="co2"/>
```

Comment:
The rest of the file uses tabs for indentation, although spaces are correct. You could invoke `mvn spotless:apply` to fix the formatting.